patroni/dcs/kubernetes.py

```diff
@@ -1263,19 +1266,28 @@ def set_config_value(self, value: str, version: Optional[str] = None) -> bool:
     def touch_member(self, data: Dict[str, Any]) -> bool:
         cluster = self.cluster
         if cluster and cluster.leader and cluster.leader.name == self._name:
-            role = 'master'
+            tmp_role = 'master'
+            role = self._leader_label_value
         elif data['state'] == 'running' and data['role'] not in ('master', 'primary'):
-            role = data['role']
+            tmp_role = data['role']
+            role = self._follower_label_value if data['role'] == 'replica' else data['role']
```

Comment:
That is, we are fine with hardcoded role=standby_leader?

Reply:
added a `standby_leader_label_value`